R/fiveyeargwsum-styles.R

```diff
@@ -5,9 +5,9 @@ getFiveyearStyle <- function(data) {
     styles <- switch(names(data), 
                      stat = list(lines = list(x=x, y=y, type="o", col="black", pch=20, cex=0.5, lwd=0.8, legend.name=data[[1]]$legend.name)),
                      est_stat = list(lines = list(x=x, y=y, col="red", lty=2, legend.name=data[[1]]$legend.name)),
-                     max_iv = list(points = list(x=x, y=y, col="red", pch=8, cex=2, legend.name=paste("Max. Instantaneous Water Level:", y)),
+                     max_iv = list(points = list(x=x, y=y, col="red", pch=8, cex=2, legend.name=paste("Min. Instantaneous (Lowest) Water Level:", y)),
```

Comment:
@dpattermann-usgs could you have it say "Min. (Lowest) Instantaneous..." instead? Same for max - just move where the word in parentheses is?